docs/tutorials/how-to-estimator.ipynb

```diff
@@ -0,0 +1,168 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a4477909",
+   "metadata": {},
+   "source": [
+    "# Get started with the Estimator primitive\n",
+    "\n",
+    "Learn how to set up and use the Estimator primitive program.\n",
+    "\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "The Estimator primitive lets you efficiently calculate and interpret expectation values of quantum operators required for many algorithms. You can specify a list of circuits and observables, then evaluate expectation values and variances for a given parameter input.  \n",
+    "\n",
+    "\n",
+    "## Prepare the environment\n",
+    "\n",
+    "1. Follow the steps in the [getting started guide](/docs/quantum-computing?topic=quantum-computing-quickstart) to get your quantum service instance ready to use.\n",
+    "\n",
+    "2. You'll need at least one circuit to submit to the program. Our examples all have circuits in them, but if you want to submit your own circuit, you can use Qiskit to create one. To learn how to create circuits by using Qiskit, see the [Circuit basics tutorial](https://qiskit.org/documentation/tutorials/circuits/01_circuit_basics.html).\n",
+    "\n",
+    "3. Create a list of observables. Observables let you define the properties of the circuit that are relevant to your problem and enable you to efficiently measure their expectation value. For simplicity, you can use the [PauliSumOp class](https://qiskit.org/documentation/stubs/qiskit.opflow.primitive_ops.html#module-qiskit.opflow.primitive_ops) in Qiskit to define them, as illustrated in the example below.\n",
+    "\n",
+    "\n",
+    "## Start a session\n",
+    "\n",
+    "With Qiskit Runtime primitives, we introduce the concept of a session or a factory that allows you to define a job as a collection of iterative calls to the quantum computer. When you start a session, it caches the data you send so it doesn't have to be transmitted to the Quantum Datacenter on each iteration.\n",
+    "\n",
+    "\n",
+    "### Specify program inputs\n",
+    "\n",
+    "The Estimator takes in:\n",
+    "* The **circuits** you want to investigate.\n",
+    "* The **parameters** input to evaluate the circuits.\n",
+    "* The **observables** (Hamiltonians) to be evaluated.\n",
+    "* Optional: The **backend** to run on. If one is not specified, the least busy backend is used. To learn about choosing a backend, see [Choose a backend](/docs/quantum-computing?topic=quantum-computing-choose-backend).\n",
```

Comment:
Broken link here as well. I guess we can just remove that line.
```suggestion
    "* Optional: The **backend** to run on. If one is not specified, the least busy backend is used.\n",
```

Reply:
Same 👆